content/exchange-guide.md

```diff
@@ -0,0 +1,575 @@
+# Listing XRP as an Exchange
+
+This document describes the steps that an exchange needs to take to list XRP. For details about other aspects of `rippled` and the Ripple Consensus Ledger (RCL), see the  [Ripple Developer Center](https://ripple.com/build).
+
+## Alpha Exchange
+
+For illustrative purposes, this document uses a fictitious business called _Alpha Exchange_ to explain the high-level steps required to list XRP. For the purposes of this document, Alpha Exchange:
+
+* Currently specializes in listing BTC/USD
+
+* Wants to add BTC/XRP and XRP/USD trading pairs
+
+* Maintains balances for all of its customers
+
+* Maintains balances for each of its supported currencies
+
+### User Benefits
+
+Alpha Exchange wants to list BTC/XRP and XRP/USD trading pairs partially because listing these pairs will benefit its users. Specifically, this support will allow its users to:
+
+* Deposit XRP _to_ Alpha Exchange _from_ the RCL
+
+* Withdraw XRP _from_ Alpha Exchange _to_ the RCL
+
+* Trade XRP with other currencies, such as BTC, USD, amongst others
+
+## Prerequisites for Supporting XRP
+
+To support XRP, Alpha Exchange must:
+
+* Create and maintain new [accounts](#accounts)
+
+* Create and maintain [balance sheets](#balance-sheets)
+
+See also:
+
+* [Gateway Compliance](https://ripple.com/build/gateway-guide/#gateway-compliance) — Gateways and exchanges are different, but exchanges should also ensure that they are complying with local regulations and reporting to the appropriate agencies.
+
+* [Requirements for Sending to RCL](https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl)
+
+* [Requirements for Sending to RCL](https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl)
```

Comment:
Replaced with: 

* [Requirements for Receiving from RCL](https://ripple.com/build/gateway-guide/#requirements-for-receiving-from-rcl)